client/src/shared/mapbox/mapbox.js

```diff
@@ -0,0 +1,17 @@
+angular.module('app').directive('mapbox', [
+   function () {
+       return {
+           restrict: 'EA',
```

Comment:
incorrect number of spaces for indentation. It looks like it's using four spaces. Should be two